src/Control/Effect/Catch/Internal.hs

```diff
@@ -5,15 +5,10 @@ module Control.Effect.Catch.Internal
 ( Catch(..)
 ) where
 
-import Control.Effect.Class
-
 -- | 'Catch' effects can be used alongside 'Control.Effect.Throw.Throw' to provide recoverable exceptions.
 --
 -- @since 1.0.0.0
 data Catch e m k
   = forall b . Catch (m b) (e -> m b) (b -> m k)
 
 deriving instance Functor m => Functor (Catch e m)
-
-instance Effect (Catch e) where
-  thread ctx handler (Catch m h k) = Catch (handler (m <$ ctx)) (handler . (<$ ctx) . h) (handler . fmap k)
```

Comment:
Not only do we no longer require the `Effect` instance, but we also don’t require the `Functor` instance. I intend to remove it in a future PR.